crates/bevy_text/src/text.rs

```diff
@@ -323,11 +328,38 @@ impl Default for TextFont {
         Self {
             font: Default::default(),
             font_size: 20.0,
+            line_height: LineHeight::default(),
             font_smoothing: Default::default(),
         }
     }
 }
 
+/// Specifies the height of each line of text for `Text` and `Text2d`
+///
+/// Default is 1.2x the font size
+#[derive(Debug, Clone, Copy, Reflect)]
```

Comment:
This is missing a `#[reflect(Debug)]` and also a `.register_type::<LineHeight>()` on the plugin.